File: discord.js
const { Client, GatewayIntentBits, Partials } = require('discord.js');
const fuzz = require('fuzzball');
const sendsms = require('./AI/test')

const token = process.env.DISCORDTOKEN; //Токен, сохраненный на 5-м шаге данного руководства 
const client = new Client({
  intents: [
    GatewayIntentBits.Guilds,
    GatewayIntentBits.GuildMessages,
    GatewayIntentBits.GuildPresences,
    GatewayIntentBits.GuildMembers,
    GatewayIntentBits.GuildMessageReactions,
    GatewayIntentBits.GuildMessageTyping, 
    GatewayIntentBits.DirectMessages,
    GatewayIntentBits.MessageContent,
  ],
  partials: [Partials.Channel]
});

var logs = true;
client.on("ready", () =>{
    console.log("Вход в бота сделан успешно"); //Сообщение, когда бот в сети 
});

client.on("messageCreate", (message) => {
    var summonUP = fuzz.partial_ratio('Петал', message.content) // Настройки вызова fuzzywuzzy для дискорда
    if (message.content == "!LogsOn") {
        logs = true 
        message.channel.reply("Логи включены!")
    }
    if (message.content == "!LogsOff") {
        logs = false
        message.channel.reply("Логи выключены!")
    }
    if (logs == true) {
        console.log("Петал, соотношение: " + summonUP);
        console.log(message.author + ": " + message.content)
    }
    if (summonUP > 80) {
        sendsms.answer(message.author, message.content)
}
});

client.login(token);

File: Discord.cs
using System;
using System.Runtime.InteropServices;
using System.Threading.Tasks;
using System.Diagnostics;
using System.Diagnostics.Tracing;
using System.IO;

using Discord;
using Discord.WebSocket;
using FuzzySharp;
using Python.Runtime;

using System.Security.Cryptography.X509Certificates;
using Discord.Audio.Streams;
using System.Data.SqlTypes;

namespace Discord_Petal
{
    
    class Program
    {   
        string token = "";
        DiscordSocketClient client;
        static void Main(string[] args) => new Program().MainAsync().GetAwaiter().GetResult();
        DiscordSocketConfig config = new DiscordSocketConfig();
        
        public async Task MainAsync()
        {
        using (Py.GIL())
        {
            dynamic test = Py.Import("neiroPetal");
        }
            config.GatewayIntents = GatewayIntents.AllUnprivileged | GatewayIntents.MessageContent;
            client = new DiscordSocketClient(config);
            client.MessageReceived += CommandsHandler;
            client.Log += Log;

            await client.LoginAsync(TokenType.Bot, token);
            await client.StartAsync(); 
            Console.ReadLine();

            await Task.Delay(-1);
        }
        private Task Log(LogMessage msg)
        {
            Console.WriteLine(msg.ToString());
            return Task.CompletedTask;
        }
        string Answer(string user, string message)
    {
        using (Py.GIL())
        {
            dynamic Petal = Py.Import("neiroPetal");
            dynamic r1 = Petal.answer(user, message); //OK
            Console.WriteLine(r1);
            return r1;     
        }
    }

        private Task CommandsHandler(SocketMessage msg)
        {
            int checksummon = Fuzz.PartialRatio("Петал", msg.Content);
            if (!msg.Author.IsBot) {
                Console.WriteLine(checksummon);
                if (checksummon > 95) {
                    msg.Channel.SendMessageAsync("dsa");

                    Console.WriteLine("test1");
                    msg.Channel.SendMessageAsync(Answer(msg.Author.Username, msg.Content));
                    Console.WriteLine("test2");
                }
            }
            return Task.CompletedTask;
        }
    }
}
// 